src/plugins/data_source/server/saved_objects/data_source_saved_objects_client_wrapper.ts

```diff
@@ -290,6 +304,7 @@ export class DataSourceSavedObjectsClientWrapper {
         savedObjects: {} as any,
         cryptography: this.cryptography,
         testClientDataSourceAttr: attributes as DataSourceAttributes,
+        request,
```

Comment:
Assuming `this.dataSourcesService.getDataSourceClient` already supports this option being passed?

Reply:
yes, it's an optional param tho